tests/v2/validation/cli/README.md

```diff
@@ -0,0 +1,24 @@
+# CLI
+
+For CLI tests, the local cluster is used to perform each of the tests. It is important to note that you must NOT have any existing downstream clusters existing in the Rancher server specified.
```

Comment:
why can't it have downstream clusters?

Reply:
Logging into the Rancher CLI, you need to specify the project you want to use. This is can only be done interactively and not silently with a flag. This is a limitation with the CLI, so we need to go through the local cluster so that there is only one project to choose from.

That will login the local cluster each time.

Reply:
The problem is there are too many interactive factors to control. Several of them are:
- There is not a clear way to determine the downstream cluster you want to choose as the project. This is an issue if you even have more than one.
- Even if you have one downstream cluster, you need to use a tool like `expect` to interactively enter in the project number you want.
     - We would have to update the Jenkinsfile to install this package as well

Along with other limitations with the CLI, it's much more feasible to run the tests on a local cluster.

Reply:
I ask because, as a user we should not be creating projects/namespaces in the local cluster. I understand these were designed to be run as an admin user but a standard user, should not be using local cluster to run these tests.

Reply:
We can probably discuss offline. These tests are specifically being ran as an admin user due to limitations with the standard user and with downstream clusters. That adds a whole other net of problems.

Reply:
Discussed offline. As of now, we have it as local cluster for now. Ideally, we would both like downstream clusters, but we're hitting a limitation with how the clients are written. It may not be possible.

I will keep looking into it, but it's a non-blocker at this time if we aren't able to get it.